app/flask/models/tasks.py

```diff
@@ -27,26 +27,47 @@ def __init__(self, conn, cursor) -> None:
             """
         )
 
+        self.cursor.execute(
+            "ALTER TABLE Tasks ADD COLUMN IF NOT EXISTS is_deleted BOOLEAN DEFAULT false;"
+        )
+
         self.conn.commit()
 
     def get_tasks(self):
-        self.cursor.execute("SELECT * FROM Tasks")
+        self.cursor.execute("SELECT * FROM Tasks WHERE is_deleted=false")
+        return self.cursor.fetchall()
+
+    def get_deleted_tasks(self):
+        self.cursor.execute("SELECT * FROM Tasks WHERE is_deleted=true")
         return self.cursor.fetchall()
 
     def add_task(self, task):
         self.cursor.execute(
-            "INSERT INTO Tasks (NAME, COMMENT) VALUES (%s, %s);",
+            "INSERT INTO Tasks (NAME, COMMENT, IS_DELETED) VALUES (%s, %s, %s);",
+            (task["name"], task["comment"], False),
```

Comment:
(task["name"], task["comment"], **False**),
by default deleted ==false, are you sure that we need to pass it here?